site/content/api/Event-API.md

```diff
@@ -76,4 +81,82 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+NB: currently there is only one such *well-known* event.
+
+### Lifecycle Notifications
```

Comment:
FMPOV this has not much to do with the lifecycle of a device (which is governed by registering the device, disabling it, etc).

Reply:
Full ack, will rename the notification - lifecycle is not a good naming.